src/components/RewardsModal/RewardsModal.tsx

```diff
@@ -0,0 +1,72 @@
+import { Button, ButtonColor } from '../Button/Button'
+import { shortenAddress } from '../../utils/shortenAddress'
+import { Modal } from '../Modal/Modal'
+import { format } from '../../utils'
+import { useEffect, useState } from 'react'
+import { getConnection } from '../../utils/useConnect'
+import { femtoKiltToDigits } from '../../utils/conversion'
+
+export interface Props {
+  accountAddress: string
+  rewards: number
+  onConfirm: () => void
+  closeModal: () => void
+}
+
+export const RewardModal: React.FC<Props> = ({
+  accountAddress,
+  rewards,
+  onConfirm,
+  closeModal,
+}) => {
+  const shortAddress = shortenAddress(accountAddress)
+  const [fee, setFee] = useState<number>(0.0003)
+
+  useEffect(() => {
+    const getFee = async () => {
+      const api = await getConnection()
+      const increment = await api.tx.parachainStaking.incrementDelegatorRewards().paymentInfo(accountAddress)
+      const claim = await api.tx.parachainStaking.claimRewards().paymentInfo(accountAddress)
+
+      if (increment.partialFee || claim.partialFee) {
```

Comment:
according to the typings this can never be undefined, and because this is a scale encoding  I doubt it will be falsy if the fee is 0 - would suspect this is always true.

Reply:
[81f2a90](https://github.com/BTE-Trusted-Entity/stakeboard/pull/74/commits/81f2a90085ffc7ecbc53129945ec28c05d9406d0)

Reply:
Or we just remove the check at all.